hedera-node/src/main/java/com/hedera/services/contracts/sources/TxnAwareEvmSigsVerifier.java

```diff
@@ -119,7 +119,7 @@ public boolean cryptoKeyIsActive(final JKey key) {
 	}
 
 	private boolean isCryptoKeyActiveInFrame(final JKey key) {
-		final var pkToCryptoSigsFn = txnCtx.accessor().getRationalizedPkToCryptoSigFn();
+		final var pkToCryptoSigsFn = txnCtx.platformTxnAccessor().getRationalizedPkToCryptoSigFn();
```

Comment:
To reflect the interface type, might also change this method name to `swirldsTxnAccessor()`.

Reply:
Modified !